WordPress/Classes/Utility/WebKitViewController.swift

```diff
@@ -190,7 +199,7 @@ class WebKitViewController: UIViewController {
 
     private func setupCloseButton() {
         let closeButton = UIBarButtonItem(image: Gridicon.iconOfType(.cross), style: .plain, target: self, action: #selector(WebKitViewController.close))
-        closeButton.accessibilityLabel = NSLocalizedString("Dismiss", comment: "Dismiss a view. Verb")
+        closeButton.accessibilityLabel = NSLocalizedString("Dismiss Webview", comment: "Dismiss a view. Verb")
```

Comment:
“Dismiss” is good enough since the context is clear. 

This is suggested [here](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/iPhoneAccessibility/Making_Application_Accessible/Making_Application_Accessible.html#//apple_ref/doc/uid/TP40008785-CH102-SW20):

> Very briefly describes the element. Ideally, the label consists of a single word, such as Add, Play, Delete, Search, Favorites, or Volume.
> Strive to design your application so that a single word identifies an element and makes its usage obvious in the current context.